install.sh

```diff
@@ -487,7 +487,7 @@ do_install_rpm() {
                     :
                     ;;
                 9) # We are currently using EL8 packages for EL9 as well
```

Comment:
```suggestion
                9)
```

Reply:
Actually we don't need this at all. Just take out the whole `9)` case and add 9 to the `7|8)` case up above.